src/client/modules/main/commands/muteMessage/MuteMessage.js

```diff
@@ -0,0 +1,51 @@
+import l10n from 'modapp-l10n';
+import Err from 'classes/Err';
+
+const usageText = 'mute message';
+const shortDesc = 'Mute messages from other characters';
```

Comment:
This wording is a bit wonky IMO because "messages" has multiple meanings, but hopefully it's clear enough in the context.

Reply:
Maybe something like "Mute global messages" or "Mute realmwide messages from other characters" could be more clear

Reply:
Yes. "messages" becomes ambiguous. But I think that you use it correctly here. It is rather that we, in other places where we use  "message" when referring to communication in general, really should use "communication". 
But for now, I think `mute message` works fine.